elliottlib/cli/find_cve_flaws_cli.py

```diff
@@ -0,0 +1,115 @@
+import bugzilla
+import click
+from errata_tool import Erratum
+from elliottlib import Runtime, bzutil
+from elliottlib.cli.common import cli, use_default_advisory_option, find_default_advisory
+from elliottlib.find_cve_flaws import get_attached_tracker_bugs, \
+    get_corresponding_flaw_bugs, is_first_fix, is_security_advisory, \
+    get_highest_security_impact, is_advisory_impact_smaller_than
+
+
+pass_runtime = click.make_pass_decorator(Runtime)
+
+
+@cli.command('find-cve-flaws',
+             short_help='Attach corresponding flaw bugs for trackers in advisory (first-fix only)')
+@click.option(
+    '--dry-run', is_flag=True,
+    help="Don't really convert to RHSA and attach flaw bugs. Just print what would be done.")
+@click.option("--add", "-a", 'advisory_id',
+              default=False, metavar='ADVISORY',
+              help="Use specified ADVISORY.")
+@use_default_advisory_option
+@pass_runtime
+def find_cve_flaws_cli(runtime, dry_run, advisory_id, default_advisory_type):
+    """Attach corresponding flaw bugs for trackers in advisory (first-fix only).
+
+    Also converts advisory to RHSA, if not already.
+
+    Example:
+
+    $ elliott --group openshift-4.6 find-cve-flaws --use-default-advisory image
+    INFO Cloning config data from https://github.com/openshift/ocp-build-data.git
+    INFO Using branch from group.yml: rhaos-4.6-rhel-8
+    INFO found 114 tracker bugs attached to the advisory
+    INFO found 58 corresponding flaw bugs
+    INFO 23 out of 58 flaw bugs considered "first-fix"
+    INFO Adding the following BZs to the advisory: [1880456, 1858827, 1880460,
+    1847310, 1857682, 1857550, 1857551, 1857559, 1848089, 1848092, 1849503,
+    1851422, 1866148, 1858981, 1852331, 1861044, 1857081, 1857977, 1848647,
+    1849044, 1856529, 1843575, 1840253]
+    """
+    runtime.initialize()
+    bzurl = runtime.gitdata.load_data(key='bugzilla').data['server']
+    bzapi = bugzilla.Bugzilla(bzurl)
+
+    if bool(default_advisory_type) == bool(advisory_id):
+        print("Exactly one of '--add' or '--use-default-advisory' options must be specified.")
+        exit(1)
+
+    if default_advisory_type is not None:
+        advisory_id = find_default_advisory(runtime, default_advisory_type)
+
+    attached_tracker_bugs = get_attached_tracker_bugs(bzapi, advisory_id)
+    # # @TODO: remove this block before merge, let it here for now because it
+    # # is useful for testing the PR, when BZs are not yet attached to an advisory
+    # attached_tracker_bugs = bzapi.query(bzapi.build_query(
+    #     product='OpenShift Container Platform',
+    #     status=['MODIFIED', 'ON_QA', 'VERIFIED'],
+    #     target_release='3.11.z',
+    #     keywords=['Security', 'SecurityTracking'],
+    # ))
```

Comment:
Is it ok to remove them now?

Reply:
yep: https://github.com/openshift/elliott/pull/158/commits/aec3f54bafc25bafdaa4968db3735a061a348af5